tests/requestparsing/README.md

```diff
@@ -0,0 +1,5 @@
+# Request Parsing tests
```

Comment:
So, we've added README.md in the `testing/` directory mostly because there will be no packages there directly, so it might give people a general idea of what to expect there.

That being said, package documentation should live in godoc, so keep it there: https://blog.golang.org/godoc. You can either create a `doc.go` file or just put it on top of one of the package files.